qutebrowser/config/configdata.yml

```diff
@@ -223,21 +224,21 @@ content.cookies.accept:
       - no-unknown-3rdparty: "Accept cookies from the same origin only, unless a
           cookie is already set for the domain."
       - never: "Don't accept cookies at all."
-  desc: Control which cookies to accept.
+  desc: Which cookies to accept.
 
 content.cookies.store:
   default: true
   type: Bool
   desc: >-
-    Store cookies.
+    Whether to store cookies.
 
     Note this option needs a restart with QtWebEngine on Qt < 5.9.
 
 content.default_encoding:
   type: String
   default: iso-8859-1
   desc: >-
-    Default encoding to use for websites.
+    The default encoding to use for websites.
```

Comment:
Same here - I don't think the "The " prefix adds anything substantial. Some things definitely sound worse than before, IMHO.